### DISCUSSION TITLE
getting undefined method Statamic\Globals\Variables::augmentable() when it should be a simple 404

### DISCUSSION THREAD

**Discussion by Ravenna** [statamic/cms#6948] [Q&A]
I have run into a weird issue where I am getting a "Call to undefined method Statamic\Globals\Variables::augmentable()" if I got to a page URL that doesn't exist. 

Example: siteurl.com/test shows this error because there is no page for that.  But if I got to a url that has a page, then it works just fine. 

Any ideas why this could happen?

**Comment by Ravenna** [comment]
So after some more looking it looks like this is coming from the SEO Pro addon.  Still digging there.

**Reply by Ravenna** [reply]
The error is coming from: vendor/statamic/seo-pro/src/Tags/SeoProTags.php(37): Illuminate\\Support\\Optional->__call('augmentable', Array)

It seems like one of two things is wrong:
1. I have not input some default data or something (even though I don't think I needed to)
2. I have something set wrong

not sure

**Comment by Ravenna** [comment]
I think this issue should be transferred over to the seo-pro repo. But GitHub won't let me do it.

**Reply by jasonvarga** [reply]
Can you just open an issue there instead please?

**Reply by Ravenna** [reply]
of course. sorry.  and thanks

**Reply by jasonvarga** [reply]
No worries, thanks!
